examples/Finding_Local_Peaks_in_Solar_Data.py

```diff
@@ -0,0 +1,76 @@
+"""
+=================================
+Finding Local Peaks in Solar Data
+=================================
+
+How to find regions of local maxima
```

Comment:
can we elaborate here? More description on what it's doing and what function it's using to do it.